package/samplers/llambo/requirements.txt

```diff
@@ -0,0 +1,10 @@
+# Core dependencies
+langchain==0.1.9
+openai==1.59.7
+pandas==1.3.5
+pynisher==0.6.4
+numpy==1.26.4
+scipy==1.15.2
+transformers==4.48.1
+tiktoken==0.6.0
+nest-asyncio==1.6.0
```

Comment:
Are they really hard constraints?
Would you be able to loosen the constraints and comment out the paper version constraints?